lib/security/NoopHasher.js

```diff
@@ -7,7 +7,10 @@ var Hasher = require('./Hasher')
 
 /***************************************************************************************************
  * @class NoopHasher
- * @description NOOP hasher (plain text)
+ * @description A NOOP "hasher" which just returns the data as the digest.
+ *              Useful as a placeholder or for when your data is kept as plain text.
+ *              However, a proper hash function such as {@link class:carbond.security.BcryptHasher}
+ *              would be more secure.
```

Comment:
i might change this to indicate that there is *no* security here and this should just be used for testing purposes